emloop/tests/api_test.py

```diff
@@ -425,3 +426,29 @@ def test_training_trace(tmpdir, caplog):
     assert loaded_yaml[TrainingTraceKeys.EXIT_STATUS] == 0
     assert start - loaded_yaml[TrainingTraceKeys.TRAIN_BEGIN] < datetime.timedelta(seconds=1)
     assert end - loaded_yaml[TrainingTraceKeys.TRAIN_END] < datetime.timedelta(seconds=1)
+
+
+def test_output_dir_deleted_rm_true(tmpdir):
```

Comment:
this does unit-test the `clean_output_dir` function but not the actual `emloop train --rm ... ` command
actually you could have saved yourself the trouble of creating your very own `ArgumentParser` since it does not matter if  you pass `parsed.rm` or a simple `True` to the function (and testing the parser is not the goal here).

Please catch me in the office and we will discuss how this can be tested properly.